cdk/lib/gatehouse.ts

```diff
@@ -213,5 +238,93 @@ export class Gatehouse extends GuStack {
 			domainName: props.domainName,
 			resourceRecord: app.loadBalancer.loadBalancerDnsName,
 		});
+
+		const vpc = GuVpc.fromIdParameter(this, 'IdentityVPC');
+		const rdsSecurityGroupRules = new SecurityGroup(
+			this,
+			'RDSSecurityGroupRules',
+			{
+				vpc: vpc,
+				allowAllOutbound: false,
+			},
+		);
+
+		const rdsSecurityGroupClients = new SecurityGroup(
+			this,
+			'RDSSecurityGroupClients',
+			{
+				vpc: vpc,
+				allowAllOutbound: false,
+				description: 'Allow access to Gatehouse DB from Clients',
+			},
+		);
+
+		rdsSecurityGroupRules.addIngressRule(
+			rdsSecurityGroupClients,
+			Port.tcp(databasePort),
+		);
+
+		const cluster = new DatabaseCluster(this, 'GatehouseDb', {
+			engine: DatabaseClusterEngine.auroraPostgres({
+				version: AuroraPostgresEngineVersion.VER_15_10,
+			}),
+			writer: ClusterInstance.serverlessV2('writer'),
+			readers: [
+				// Scale reader instance with writer so that it can deal with immediate traffic spike during failover
+				ClusterInstance.serverlessV2('reader', { scaleWithWriter: true }),
+			],
+			credentials: Credentials.fromPassword(
+				'postgres',
+				// This value will be replaced by the escape hatch to use ManagerMasterUserPassword below
+				SecretValue.secretsManager('uselessWorkaroundSecret'),
+			),
+			storageEncrypted: true,
+			deletionProtection: true,
+			iamAuthentication: true,
+			enableDataApi: true,
+			enableClusterLevelEnhancedMonitoring: true,
+			enablePerformanceInsights: true,
+			// Under some scenarios AWS can upgrade the database version without downtime
+			// However all upgrades, including minor ones, can result in downtime in certain scenarios.
+			// https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/USER_UpgradeDBInstance.PostgreSQL.MinorUpgrade.html
+			autoMinorVersionUpgrade: false,
+			performanceInsightRetention: PerformanceInsightRetention.DEFAULT,
+			monitoringInterval: Duration.minutes(1),
+			defaultDatabaseName: 'gatehouse',
```

Comment:
Started using Gatehouse naming, but perhaps we want to stick with `identity` or `identity_v2`?

Reply:
our current database name is `identitydb` i think

so could get confusing if we use a similar name

gatehouse isnt an ideal name either

using the `_v2` modifier isn't ideal either, as once we've turned off the old db then the v2 moniker doesn't mean anything anymore 

so yeah, i have no suggestions